types/wlr_screencopy_v1.c

```diff
@@ -48,9 +48,20 @@ static void screencopy_damage_accumulate(struct screencopy_damage *damage) {
 		return;
 	}
 
-	pixman_region32_union(region, region, &output->pending.damage);
-	pixman_region32_intersect_rect(region, region, 0, 0, output->width,
-		output->height);
+	// If the compositor did not submit damage but did submit a buffer,
+	if (!(output->pending.committed & WLR_OUTPUT_STATE_DAMAGE) &&
+		(output->pending.committed & WLR_OUTPUT_STATE_BUFFER)) {
+		// damage everything
+		int width, height;
+		wlr_output_transformed_resolution(output, &width, &height);
+		pixman_region32_union_rect(region, region, 0, 0, width, height);
+	} else {
+		// otherwise copy over the current damage
+		pixman_region32_union(region, region, &output->pending.damage);
+		pixman_region32_intersect_rect(region, region, 0, 0, output->width,
+			output->height);
```

Comment:
Looking at it now, it sounds like this breaks when using an output transform? We should use `wlr_output_transformed_resolution` like above, I think.

Can you fix that in a different commit (makes bisecting easier if we find bugs about it)?